tests/integration/test_debug.py

```diff
@@ -264,6 +267,32 @@ def test_tracer_info_level_log(self):
         assert tracer.log.log.mock_calls == []
 
 
+def test_runtime_metrics_enabled_via_manual_start():
+    f = debug.collect(ddtrace.tracer)
+    assert f.get("runtime_metrics_enabled") is False
+
+    RuntimeMetrics.enable()
+    f = debug.collect(ddtrace.tracer)
+    assert f.get("runtime_metrics_enabled") is True
+
+    RuntimeMetrics.disable()
+    f = debug.collect(ddtrace.tracer)
+    assert f.get("runtime_metrics_enabled") is False
+
+
+def test_runtime_metrics_enabled_via_env_var_start(monkeypatch, ddtrace_run_python_code_in_subprocess):
+    monkeypatch.setenv("DD_RUNTIME_METRICS_ENABLED", "true")
+    _, _, status, _ = ddtrace_run_python_code_in_subprocess(
+        """
+import ddtrace
+from ddtrace.internal import debug
+f = debug.collect(ddtrace.tracer)
+assert f.get("runtime_metrics_enabled") is True
+""",
+    )
+    assert status == 0
```

Comment:
Not a strong opinion to make this change, but figured also testing the default with no env set might be good.

```suggestion
    # default, no env variable set
    _, _, status, _ = ddtrace_run_python_code_in_subprocess(
        """
import ddtrace
from ddtrace.internal import debug
f = debug.collect(ddtrace.tracer)
assert f.get("runtime_metrics_enabled") is False
""",
    )
    assert status == 0

    # Explicitly set env variable
    monkeypatch.setenv("DD_RUNTIME_METRICS_ENABLED", "true")
    _, _, status, _ = ddtrace_run_python_code_in_subprocess(
        """
import ddtrace
from ddtrace.internal import debug
f = debug.collect(ddtrace.tracer)
assert f.get("runtime_metrics_enabled") is True
""",
    )
    assert status == 0
```

Reply:
This kind of happens with `test_standard_tags` already, however the difference with this suggestion is also testing the default with `ddtrace-run` vs just importing `ddtrace`